v2/pkg/engine/plan/abstract_selection_rewriter_test.go

```diff
@@ -46,12 +46,17 @@ func TestInterfaceSelectionRewriter_RewriteOperation(t *testing.T) {
 				break
 			}
 		}
+		require.NotEqual(t, ast.InvalidRef, fieldRef)
+
+		// Schema parses just the schema without merging with the base schema.
+		// In fact, rewriter does not use def from the FakeFactory.
+		ds := testCase.dsBuilder.
```

Comment:
to not reinit ds here, we could use schema in each test case so that rewriter will start getting it from ds configuration